test/verify/check-networkmanager-basic

```diff
@@ -42,6 +42,12 @@ class TestNetworkingBasic(NetworkCase):
         b.wait_in_text("#networking-graphs", "Transmitting")
         b.wait_in_text("#networking-graphs", "Receiving")
 
+        if not m.ostree_image:
+            # screenshot assumes running firewalld and absent virbr0 (in particular, *3* interfaces)
+            m.execute("systemctl start firewalld; virsh net-destroy default 2>/dev/null || true")
+            b.wait_not_present("tr[data-interface=virbr0]")
+            self.wait_onoff("#networking-firewall-summary", True)
```

Comment:
I'm not sure how I feel about this...

If we take extra steps just for the purpose of making sure the pixels look good, then it seems like those ought to be conditionalized on if we're actually running the pixels tests.

But I think a better way to look at it is that it's wrong that we rely on the state of a previously-running test because it's wrong that that test left that state: it's a case of non-nondestructiveness, which ought to be fixed.

Reply:
Right, the line between "leave the VM broken" and "don't make overly harsh assumptions about the VM" are blurry. However, here I feel that it's the test's responsibility to set up the conditions that it needs. Especially for virbr0, as that's such a dynamic thing -- it's equally valid and plausible for it to be up or down (a simple `virsh net-list` will bring it up, due to socket activations and daemon autostart!). So in this case my gut feeling went for this approach, which will also help us with gating (where the condition of the testbed is not strictly defined either)